api/api/authentication.py

```diff
@@ -123,9 +135,22 @@ def generate_keypair():
     return private_key, public_key
 
 
-def change_keypair():
-    """Generate key files to keep safe."""
-    key_obj = ec.generate_private_key(ec.SECP521R1())
+def change_keypair(curve: ec.EllipticCurve) -> Tuple[str, str]:
```

Comment:
Same as the second item from above

Reply:
Same reason as in https://github.com/wazuh/wazuh/pull/25097#discussion_r1718641130